classes/assets/CccReducer.php

```diff
@@ -58,16 +58,21 @@ public function reduceCss($cssFileList)
         $version = Configuration::get('PS_CCCCSS_VERSION');
         $cccFilename = 'theme-' . $this->getFileNameIdentifierFromList($files) . $version . '.css';
         $destinationPath = $this->cacheDir . $cccFilename;
+        $destinationUri = $this->getFQDN() . $this->getUriFromPath($destinationPath);
 
         if (!$this->filesystem->exists($destinationPath)) {
             CssMinifier::minify($files, $destinationPath);
         }
+        if (Tools::hasMediaServer()) {
+            $relativePath = _THEMES_DIR_ . _THEME_NAME_ . '/assets/cache/' . $cccFilename;
+            $destinationPath = $destinationUri = Tools::getCurrentUrlProtocolPrefix() . Tools::getMediaServer($relativePath) . $relativePath;
```

Comment:
Weird to have path and uri similar.. Something must be wrong here :thinking:

Reply:
In template, only the URI & media is used.

When we used a CDN/Media Server, the URI is the path where the file is.

Reply:
In *our* templates :thinking: 
A URI = http://mydomain.fr/path/to/something.jpg
A path = /path/to/something.jpg

If your path integrate the scheme + authority, it's no longer a path

Reply:
But you are totally right. I fix it now.